packages/core/content-type-builder/package.json

```diff
@@ -28,24 +28,35 @@
   },
   "dependencies": {
     "@sindresorhus/slugify": "1.1.0",
+    "@strapi/design-system": "1.6.3",
     "@strapi/generators": "4.6.1",
     "@strapi/helper-plugin": "4.6.1",
+    "@strapi/icons": "1.6.3",
     "@strapi/utils": "4.6.1",
+    "axios": "1.2.2",
     "fs-extra": "10.0.0",
+    "immer": "9.0.19",
     "lodash": "4.17.21",
+    "match-sorter": "^4.0.2",
     "pluralize": "^8.0.0",
+    "prop-types": "^15.7.2",
+    "qs": "6.11.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-helmet": "^6.1.0",
     "react-intl": "6.2.8",
     "react-redux": "8.0.5",
     "react-router": "^5.2.0",
     "react-router-dom": "5.3.4",
     "redux": "^4.2.1",
     "reselect": "^4.0.0",
+    "styled-components": "5.3.3",
     "yup": "^0.32.9"
   },
   "devDependencies": {
-    "@testing-library/react": "12.1.4"
+    "@testing-library/react": "12.1.4",
+    "@testing-library/react-hooks": "8.0.1",
+    "history": "^4.9.0"
```

Comment:
Why is history in devDependencies? Isn't that linked to `react-router-dom`?

Reply:
you don't need to install `history` for `react-router-dom` in our context here it's just used as a `devDep` for tests.

Reply:
Why do the tests need it?

Reply:
They do stuff like `import { createMemoryHistory } from 'history';`

Reply:
I can solve this actually.